polymod/Polymod.hx

```diff
@@ -480,7 +480,7 @@ class Polymod
 					if (apiScore < PolymodConfig.apiVersionMatch)
 					{
 						Polymod.error(VERSION_CONFLICT_API,
-							'Mod "$origDir" was built for incompatible API version ${meta.apiVersion)}, current version is "${apiVersion}"', SCAN);
+							'Mod "$origDir" was built for incompatible API version ${meta.apiVersion}, current version is "${apiVersion}"', SCAN);
```

Comment:
This fix looks correct. I do not know why this built on my machine without this change (maybe I fixed the issue and forgot to commit it myself?).